modules/402-ingress-nginx/webhooks/validating/ingressnginxcontroller

```diff
@@ -64,6 +64,17 @@ function __main__() {
     fi
   fi
 
+  # Prevent adding annotations for Pods with istio.io, deckhouse.io, or kubernetes.io prefixes
+if context::jq -e -r '.review.request.object.kind == "IngressNginxController"' >/dev/null 2>&1; then
+  # exists spec.controllerPodsAdditionalAnnotations
+  if context::jq -e -r '.review.request.object.spec.controllerPodsAdditionalAnnotations' >/dev/null 2>&1; then
+    if context::jq -e -r '.review.request.object.spec.controllerPodsAdditionalAnnotations | keys[] | test("(istio\\.io|deckhouse\\.io|kubernetes\\.io)")' >/dev/null 2>&1; then
```

Comment:
```suggestion
  if context::jq -e -r '.review.request.object.spec.controllerPodsAdditionalAnnotations // {} | keys[] | test("(\\.?istio\\.io/|\\.?deckhouse\\.io/|\\.?kubernetes\\.io/)")' >/dev/null 2>&1; then
```

jq trick and '/' also important to prevent smth like 'betterthanistio.ionosphere.io'.